packages/dashboard-backend/src/devworkspace-client/services/api/serverConfigApi.ts

```diff
@@ -95,4 +95,18 @@ export class ServerConfigApi implements IServerConfigApi {
       );
     }
   }
+
+  async getRunningWorkspacesLimit(): Promise<number> {
+    try {
+      const cheCustomResource = await this.getCheCustomResource();
+
+      return cheCustomResource.spec.server.runningWorkspacesLimit || 1;
```

Comment:
```suggestion
      return cheCustomResource.spec.devWorkspace.runningLimit || 1;
```

Reply:
I have fixed it.

Reply:
https://github.com/eclipse-che/che-dashboard/pull/475/commits/f74c1edb213caef3ca5a94e8334286febff0611d